timesketch/lib/experimental/create_graph.cql

```diff
@@ -32,6 +32,8 @@ WHERE line.src <> line.dst
   MERGE (src)-[r:ACCESS]->(dst)
```

Comment:
What about adding es_index to nodes also?

Reply:
Done.